resources/lang/en/patrols/forest/hunting/any.json

```
@@ -2875,7 +3639,10 @@
                         "cats_from": ["patrol"],
                         "mutual": false,
                         "values": ["like", "comfort"],
```

```suggestion
                        "values": ["comfort"],
```
just to make it consistent with the other instance of this